src/app/shared/directives/disable-browser-zoom.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, HostListener } from '@angular/core';
+
+@Directive({
+  selector: '[appDisableBrowserZoom]',
+})
+export class DisableBrowserZoomDirective {
+  @HostListener('wheel', ['$event'])
+  public onWheel(event: MouseEvent): void {
+    if (event.ctrlKey === true) {
```

Comment:
Please check can we change  
if (event.ctrlKey === true) 
to 
if (event.ctrlKey)